src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -503,14 +488,6 @@ private function addSubForm(FormBuilderInterface $builder, $name, ChoiceView $ch
 
     private function createChoiceListView(ChoiceListInterface $choiceList, array $options)
```

Comment:
It should be safe since the call is from a private method.

Even if someone codes its own factory or decorator and extends `ChoiceType`, he would have to implement its own method too. at this level he should know the internal and get it over with, so I don't think this break BC.